### PR TITLE
#35 Test create and publish a release with automatic semantic versioning

### PR DESCRIPTION
**Pull request by Zero3141**
Partly fixes [#35](https://github.com/secureCodeBox/defectdojo-client-java/issues/35)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -4,36 +4,81 @@
...
+
+      - name: Release new version
+        run: |
+          git config user.name 'github-actions[bot]'
+          git config user.email 'github-actions[bot]@users.noreply.github.com'
+          ./gradlew release
+
+      - name: Get current version
+        id: version
+        run: echo "::set-output name=version::$(./gradlew -q -Prelease.quiet currentVersion)"
```
Comment (Weltraumschaf) [inline .github/workflows/release.yml:49]: What does this?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -6,19 +6,24 @@
...
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: adopt
+
+      - name: Setup Gradle
```
Comment (Weltraumschaf) [inline .github/workflows/test.yml:25]: Why here no Gradle Validation?

**Review by Weltraumschaf — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (Zero3141) [reply .github/workflows/release.yml:51]: Added a comment before this step to explain its purpose & functionality. Is this sufficient?

Comment (Weltraumschaf) [reply .github/workflows/release.yml:50]: That's a good starting point for the reason, but there should also be some info about the concrete syntax (`::set-output name=version::`). If you have a link which describes this, a link with a hint that it describes the syntax is sufficient.

**Review by Weltraumschaf — APPROVED** [review]

**Pull request merged**